src/main/resources/assets/vampirism/lang/en_US.lang

```diff
@@ -356,6 +352,7 @@ effect.vampirism.saturation=Saturation
 effect.vampirism.sanguinare=Sanguinare Vampiris
 effect.vampirism.fire_protection=Fire Protection
 effect.vampirism.garlic=Garlic
+effect.vampirism.poison=Poison
```

Comment:
Maybe change the name to make it clear that this is not the Vanilla poison.
Maybe "Vampirism poison" or "Toxicant"